web/src/markdown.js

```diff
@@ -686,13 +686,12 @@ export function get_parsed_data(raw_content) {
     };
 }
 
-export function add_topic_links(message) {
-    if (message.type !== "stream") {
-        message.topic_links = [];
-        return;
+export function get_topic_links_for_web(topic) {
+    if (topic === undefined) {
+        return [];
     }
```

Comment:
Do we need this wrapper around `get_topic_links`?

Reply:
Sure!

Reply:
Actually can you clarify what you're expecting for `get_linkifier_map`? Would you expect `get_topic_links` to take an optional `get_linkifier_map` and then default to `web_app_helpers.get_linkifier_map`? There's something nice about being more explicit about when we use `web_app_helpers`, which this wrapper function helps with.

Reply:
The purpose of `web_app_helpers` is just to configure shared code with the React Native mobile apps, which never used this interface are deprecated, so there's not much value to avoiding inlining it. Might be worth a #frontend thread about whether we should undo some of the work done to try to make the Markdown processor shareable, but my theory is we should do so to make this code clean.

Reply:
Ohhh I see, you're suggesting `get_topic_links` not take `get_linkifier_map` and just reference `web_app_helpers.get_linkifier_map` directly. Okay yeah I can change that.